CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.1.0
+* Added React Native macOS debugging support: [more info](https://github.com/microsoft/vscode-react-native#react-native-for-macos) [#1409](https://github.com/microsoft/vscode-react-native/pull/1409)
+* Added support for `openExpoQR` debugging argument determining whether to open a tab with a QR code after launching the Expo server or not. Enhanced the workflow for Expo server starting [#1413](https://github.com/microsoft/vscode-react-native/pull/1413)
+* Fixed launch scenarios for iOS direct debugging [#1430](https://github.com/microsoft/vscode-react-native/pull/1430)
+* Values in the `target` debug scenario argument are no longer replaced by UDID in cases the existing simulator name is already used there [#1431](https://github.com/microsoft/vscode-react-native/pull/1431)
```

Comment:
```suggestion
* Values in the `target` debug scenario argument are no longer replaced by UDID in case the existing simulator name is already used there [#1431](https://github.com/microsoft/vscode-react-native/pull/1431)
```